src/stores/store/eth1/local/local.go

```diff
@@ -406,6 +418,16 @@ func getEIP712EncodedData(typedData *core.TypedData) (string, error) {
 	return fmt.Sprintf("\x19\x01%s%s", domainSeparatorHash, typedDataHash), nil
 }
 
+// getEIP191EncodedData encodes the given message that can be later recovered
+// with the given validator. following EIP-191 spec version 0x0
+func getEIP191EncodedData(address string, msg []byte) (eipFormat, msgHash []byte, err error) {
+	hexAddress, err := hex.DecodeString(address)
+	if err != nil {
+		return nil, nil, err
+	}
+	return append([]byte{0x19, 0x00}, append(hexAddress, msg...)...), crypto.Keccak256(msg), nil
```

Comment:
As far as I understand we are using a `personal_sign` so we should be appending 0x45 instead of 0x00

![image](https://user-images.githubusercontent.com/2728080/127999467-325d75bf-2a89-48c1-b7af-0fe7fb069d8c.png)

Reply:
After doing another research I think you are right and we could use `0x00`